keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/plugins/VerificationEvaluator.kt

```diff
@@ -0,0 +1,12 @@
+package com.netflix.spinnaker.keel.api.plugins
+
+import com.netflix.spinnaker.keel.api.Verification
+import com.netflix.spinnaker.keel.api.verification.VerificationStatus
+
+interface VerificationEvaluator<VERIFICATION: Verification> {
```

Comment:
Docs?

Reply:
Also, if we're planning to make this part of the exposed plugin interfaces, might as well mark it as `SpinnakerExtensionPoint`.